server/sonar-server/src/main/java/org/sonar/server/edition/FinalizeEditionChange.java

```diff
@@ -76,26 +78,43 @@ private void failIfLicenseCommitIsPresent() {
   }
 
   private void finalizeInstall(EditionManagementState.PendingStatus status) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "status". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)